tsconfig.json

```diff
@@ -5,17 +5,7 @@
     "paths": {
       "@phlare/*": ["./public/app/*"],
 
-      "@pyroscope/webapp/*": ["./node_modules/pyroscope-oss/webapp/*"],
-      "@webapp/redux/*": [
-        "./node_modules/pyroscope-oss/webapp/javascript/redux/*"
-      ],
-      "@pyroscope/flamegraph/*": [
-        "./node_modules/pyroscope-oss/packages/pyroscope-flamegraph/*"
-      ],
-      "@pyroscope/models/*": [
-        "./node_modules/pyroscope-oss/packages/pyroscope-models/*"
-      ],
```

Comment:
Note these refer to the `packages` dir, I think they still need to be aliased? Or does the `workspaces` bit in package.json handles that somehow?

Reply:
> Or does the workspaces bit in package.json handles that somehow?

Yes, that appears to be the case. But I am getting the sense that some of the aliases are clashing with the workspaces in certain circumstances. I will continue trying things.

>Don't forget to update [scripts/webpack/webpack.common.ts too!](https://github.com/grafana/pyroscope/blob/next/scripts/webpack/webpack.common.js) too!

Thanks for the note. However, my attempts to fix `webpack.common.ts` are meeting with failure locally.
I am thinking some old "node_modules" imports are still cached, so the current "passed tests" might be a misleading positive.

Reply:
I think I will get rid of the "workspace" entries, and just use aliases at this stage.